prediction_market_agent/db/blockchain_transaction_fetcher.py

```
@@ -0,0 +1,89 @@
+
+class BlockchainTransactionFetcher:
+    MIN_TRANSACTION_AMOUNT = xDai(0.001)
+
+    def __init__(self):
+        self.blockchain_table_handler = BlockchainMessageTableHandler()
+
+    def unzip_message_else_do_nothing(self, data_field: str) -> str:
+        """We try decompressing the message, else we return the original data field."""
+        try:
+            return decompress_message(HexBytes(data_field))
+        except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)